src/pin-field/PinField.stories.ts

```diff
@@ -97,6 +100,50 @@ export const Interactive: ComponentStoryFormat<Args> = {
     }
 };
 
+export const Max_Length: ComponentStoryFormat<Args> = {
+    render: (args: Args) => html`
+    <omni-pin-field
+      data-testid="test-pin-field"
+      label="${ifNotEmpty(args.label)}"
+      value="${args.value}"
+      maxLength=${args.maxLength}>
+    </omni-pin-field>
+  `,
+    name: 'Max Length',
+    description: 'Set html attribute to limit the characters of the field.',
+    args: {
+        label: 'Max Length',
+        maxLength: 5
+    },
+    play: async (context) => {
+        const pinField = within(context.canvasElement).getByTestId<PinField>('test-pin-field');
+        pinField.value = '';
+
+        const interactions = jest.fn();
+        pinField.addEventListener('input', interactions);
+
+        const inputField = pinField.shadowRoot?.getElementById('inputField') as HTMLInputElement;
+
+        await userEvent.type(inputField, '12345678910', {
+            pointerEventsCheck: 0
+        });
+        const value = 12345;
+
+        // TODO: Fix race conditions in tests
+        if (navigator.userAgent === 'Test Runner') {
+            console.log('CICD Test - Not Visual');
```

Comment:
This implies that the "assert" part of the test never actually happens via the CI/CD action. 

@chromaticWaster and @BOTLANNER please take note of this, we need to address ASAP.

Reply:
Changes merged as part of the following PR https://github.com/capitec/omni-components/pull/105